### PR TITLE
Fix qa_automation tests

### PR DESCRIPTION
**Pull request by coolo**
Do not remove SLES and SDK and readd it - we rely on fake_scc now

### CODE REVIEW

**Comment by coolo** [comment]
https://tortuga.suse.de/tests/393

File: tests/qa_automation/qa_run.pm [hunk 1]
```diff
@@ -80,16 +70,17 @@ sub run() {
 
     # When finished, the screen will terminate
     for (1 .. 60) {
-        my $ret = check_screen [qw/qa_screen_done qa_error/], 120;
-        if ($ret && match_has_tag('qa_error')) {
-            die "run failed";
-        }
-        elsif ($ret) {
+        if (check_screen [qw/qa_screen_done qa_error/], 120) {
+            if (last_match_has_tag('qa_error')) {
```
Comment (aaannz) [inline tests/qa_automation/qa_run.pm]: Do we have `last_match_has_tag`? Did I miss something

**Comment by coolo** [comment]
we don't - and I wonder it works :)

File: tests/qa_automation/qa_run.pm [hunk 2]
```diff
@@ -80,16 +70,17 @@ sub run() {
...
+            if (match_has_tag('qa_error')) {
+                die "run failed";
+            }
             last;
         }
-        # change the screen periodically to avoid standstill detection for long running tests
-        my $time = time;
-        type_string "$time\n";
+        else {
+            # change the screen periodically to avoid standstill detection for long running tests
```
Comment (okurz) [inline tests/qa_automation/qa_run.pm]: I thought we don't have a "standstill" detection anymore?

Comment (coolo) [inline tests/qa_automation/qa_run.pm]: we don't, but we still have screensavers, so it still makes somewhat sense for the video :)

**Pull request merged**